launchcontainer/launchcontainer.py

```diff
@@ -55,6 +52,8 @@
         'js_class': 'LaunchContainerXBlock'
     }
 }
+DEFAULT_SUPPORT_URL = '/help'
```

Comment:
Do we have customer docs to explain how this will work?  "By default, your support URL will be <yourOpenEDXorTahoeDomain>/help.  Please configure the URL at X location in your settings."  I'm wondering about the case where a site does not have a support page.  Perhaps having some mechanism to not post the support URL if it's blank?

Reply:
Thanks @briandant - This makes me think that the best thing is to actually put help text on the support_email field which explains that an email overrrides a support url, and tells what the support_url is set to.  I'll add a commit with that to this PR.